pkg/common/cns-lib/node/manager.go

```diff
@@ -63,6 +63,8 @@ type Manager interface {
 	GetAllNodes(ctx context.Context) ([]*vsphere.VirtualMachine, error)
 	// UnregisterNode unregisters a registered node given its name.
 	UnregisterNode(ctx context.Context, nodeName string) error
+	// GetNodeNameByVmMoID fetches the k8s name of the given node given the VM MoID
+	GetNodeNameByVmMoID(ctx context.Context, vmMoID string) (string, error)
```

Comment:
Hi @smanjunath3 , Why would you require the node Name?

Reply:
@BaluDontu as per the design document for ListVolume API, we need this to populate the response for ListVolume. Please refer to https://confluence.eng.vmware.com/display/CNAS/Design+Specification+-+CSI+List+Volumes+API#DesignSpecificationCSIListVolumesAPI-Vanilla.1 under ListVolumes API for Vanilla
step 11.